app/src/main/java/com/doubtless/doubtless/screens/doubt/HomeMainScreenFragment.kt

```diff
@@ -38,31 +39,39 @@ class HomeMainScreenFragment : Fragment() {
         if (_navigator != null) navigator = _navigator
 
         viewModel = getViewModel()
-        viewModel.fetchTags()
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        viewModel.fetchTags()
```

Comment:
if there is already data fetched then dont fetch again, if (viewmodel.result.data != null) something like this